role-service/src/main/java/uk/ac/ox/ndph/mts/role_service/controller/RoleController.java

```diff
@@ -59,7 +59,7 @@ public Role getOne(@PathVariable String id) throws ResponseStatusException {
     }
 
     @GetMapping(params = "ids")
-    @PreAuthorize("@authorisationService.authoriseUserPermissionRoles(#ids)")
+    @PreAuthorize("@authorisationService.authUserPermRoles(#ids)")
```

Comment:
general tidy as I wanted to make the methods less verbose to allow checkstyle to pass for line length